app/services/dqt/client.rb

```diff
@@ -0,0 +1,26 @@
+module Dqt
+  class Client
+    class Request
+      include HTTParty
+      base_uri ENV.fetch("DQT_BASE_URL", "")
+      headers "Accept" => "application/json",
+              "Content-Type" => "application/json;odata.metadata=minimal",
+              "Authorization" => "Bearer #{ENV.fetch("DQT_API_KEY", "")}",
+              "X-Api-Version" => ENV.fetch("DQT_API_MINOR_VERSION", "20240101")
```

Comment:
Yes, the docs say it's required.

Reply:
Sorry, I mean in terms of the ENV var name. Is `DQT_API_VERSION` as sufficient ENV var name?

I don't know if I would _personally_ call `20240101` a _minor_ version :/ It looks to be just a "version" based on the date. Major-Minor-Patch of semver comes to mind with the terminology of "minor" version. But I understand that is how it is defined in the Teaching Record docs.

Reply:
I think we should refer to it as it is referred to in the docs, otherwise we have to explain somewhere that what we are calling DATED_VERSION (or similar) is what is referred to as Minor version in the docs.

Reply:
Also the version is actually v3. So this date is not the version exactly -- it's a subversion. Or, as the docs refer to it 'minor' version.